src/main/frontend/config.cljs

```diff
@@ -63,49 +64,20 @@
     (if dev? path
         (str asset-domain path))))
 
-(defn text-formats
-  []
-  (let [config-formats (some->> (get-in @state/state [:config :text-formats])
```

Comment:
I remove this state from these format fns since they were introduced in 2020 and didn't seem to be used. Supporting this would take some additional work as `:config` would need to be a separate atom that gets migrated to `gp-config`